src/util/apis/generated/operations/index.d.ts

```diff
@@ -13849,6 +13852,9 @@ export interface ReleaseUploads {
      *
      * @param {number} [options.releaseId] The ID of the release.
      *
+     * @param {string} [options.buildVersion] The build version of the uploaded
+     * WPF/Winforms
```

Comment:
to align with `releases` swagger, let's say `The build version of the uploaded binary`

Reply:
Thanks for heads up, missed that we updated the text. Updated swagger file used and regenerated models